utils/sysinfo/sysinfo.go

```diff
@@ -212,10 +212,10 @@ func GetNodesInfo(sysFs sysfs.SysFs) ([]info.Node, int, error) {
 		node.Cores = cores
 
 		allLogicalCoresCount += logicalCoreCount
-		err = addCacheInfo(sysFs, &node)
-		if err != nil {
-			return nil, 0, err
-		}
+
+		// On some Linux platforms(such as Arm64 guest kernel), cache info may not exist.
+		// So, we should ignore error here.
+		_ = addCacheInfo(sysFs, &node)
```

Comment:
Can we `klog.Warningf` here at least?  It will log frequently on Arm, but this is just meant to fix the conformance test temporarily.

Reply:
Done.